src/web-ui/src/partials/AppModal/DemoGuide/DemoGuideArticle/articles/AmazonPayIntegration.vue

```diff
@@ -0,0 +1,129 @@
+<template>
+  <ArticleLayout>
+    <template #heading>Amazon Pay Integration</template>
+    <p>
+      Amazon Pay is one of many online payment services that offer low friction,
+      secure checkout across different platforms. There is a partial integration of
+      <a href="sellercentral.amazon.com/">Amazon Pay for merchants</a>
+      into the Retail Demo Store.
+    </p>
+    <p>
+      If you enter your Amazon Pay credentials as parameters into CloudFormation when
+      deploying or updating this demo, an Amazon Pay button will be generated for you,
+      signed with those credentials. This Amazon Pay button allows users to intitate a
+      checkout session with your Amazon Pay store as the seller.
+    </p>
+
+    <div class="popout">
+        <span class="note-head">Note:</span>
+        <span class="note-body"> This is a partial integration that does not cover the full checkout process.
+          We go as far as initiating a checkout session which you can test by logging in with a sandbox user.</span>
+    </div>
+
+    <ArticleFeature>
+      <template #title>How to Set up Amazon Pay Integration</template>
+      <template #default>
+        <p>
+          In order to set up the integration so that you can initiate a checkout session, we need to set up an
+          Amazon Pay merchant account, as well as a store, and obtain the integration credentials for that store. Having
+          obtained these credentials we can supply them to Retail Demo Store so that it can start a credentialed
+          checkout session which you can enter by authenticating a test user. For more information about integrating
+          with Amazon Pay see <a href="https://developer.amazon.com/docs/amazon-pay-checkout/get-set-up-for-integration.html">
+          these instructions</a>.
```

Comment:
These step by step instructions in this file sound more appropriate for the repo than in the demo guide. The demo guide is intended for someone receiving a demo and to better understand Amazon Pay and how to experience it in the demo. We're not trying to explain how they'd set up and deploy the RDS with Pay.